### PR TITLE
Add test for the docker images we build

### PR DESCRIPTION
**Pull request by svenklemm**

### CODE REVIEW

File: .github/workflows/docker-images.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Test our docker images are built with the most recent version
+name: Docker images
```
Comment (jnidzwetzki) [inline .github/workflows/docker-images.yaml:2]: Nit: 

```suggestion
name: Test Docker images
```

To use the same naming as we use in other tests like "Test Update and Downgrade"

File: .github/workflows/docker-images.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        psql -c "SELECT extname,extversion,version() FROM pg_extension WHERE extname='timescaledb'"
+        installed_version=$(psql -X -t \
+          -c "SELECT extversion FROM pg_extension WHERE extname='timescaledb';" | sed -e 's! !!g')
+        if [ "${{ steps.versions.outputs.version }}" != "$installed_version" ];then
+          false
+        fi
+
+    - name: Smoke test
+      run: |
+        psql -c "CREATE TABELE metrics(time timestamptz, device text, metric text, value float);"
```
Comment (jnidzwetzki) [inline .github/workflows/docker-images.yaml:72]: Have you considered adding a `create_hypertabe` call as well to invoke the extension code?

**Review by jnidzwetzki — APPROVED** [review]
Added two comments, rest looks good to me.

**Review by mkindahl — COMMENTED** [review]
This seems to test the docker images but do not really need anything from the repository. Shouldn't we rather have this test in https://github.com/timescale/timescaledb-docker?

**Comment by svenklemm** [comment]
> This seems to test the docker images but do not really need anything from the repository. Shouldn't we rather have this test in https://github.com/timescale/timescaledb-docker?

This does not only test images built by timescaledb-docker. I think having all the packaging tests in the main repo is convenient, all the packaging tests for the other platforms are in the main repo as well.

**Pull request merged**